airsenal/framework/utils.py

```diff
@@ -207,15 +207,19 @@ def get_sell_price_for_player(
     if the price increased in that time, we only get half the profit.
     if gameweek is None, get price we could sell the player for now.
     """
-    if not fpl_team_id:
-        fpl_team_id = fetcher.FPL_TEAM_ID
+    if fpl_team_id:
```

Comment:
I think you're not actually using this function any more, and I actually don't think it's used anywhere in the codebase now (using the one in the `Squad` class instead). So maybe we could just delete it.

As a separate issue (not for this PR), maybe we should make all optmisation/transfer related scripts query the API for the latest status of a squad in the API before running, rather than relying on `airsenal_update_db` having been run recently.